src/init.c

```diff
@@ -345,6 +352,8 @@ shmem_internal_init(int tl_requested, int *tl_provided)
             printf("SMA_CMA_PUT_MAX         %s\n", shmem_transport_cma_put_max);
             printf("SMA_CMA_GET_MAX         %s\n", shmem_transport_cma_get_max);
 #endif /* USE_CMA */
+            printf("SMA_DEBUG               %s\n", (NULL != shmem_util_getenv_str("DEBUG")) ? "On" : "Off");
+            printf("\tValue for runtime debug information.\n");
```

Comment:
This description is a little odd.  Could you make it something like "Enable debugging messages".